controllers/shooter.py

```diff
@@ -164,15 +177,14 @@ def firing(self, initial_call) -> None:
                     self.vision.target_yaw,
                 ]
             )
-        if self.flywheels_running:
-            if self.interpolation_override:
-                self.shooter.motor_speed = self.flywheel_speed
-            else:
-                self.shooter.motor_speed = interpolate(
-                    self.distance, self.ranges_lookup, self.flywheel_speed_lookup
-                )
+        self._track()
```

Comment:
Looks like this will adjust the flywheel speeds if we start to fire whilst we're moving. Is this intended?

Reply:
Yes, I think we should have the most up to date range and bearing info for our shots right up to the moment it comes out the turret.

Reply:
And if not, those statements were redundant anyway because the stored values were never changed in that state.

Reply:
Doesn't the shooter speed reset to 0 if it's not commanded each control loop?